backend/typescript/types.ts

```diff
@@ -1,12 +1,12 @@
-export type Role = number;
+export type Role = "User" | "Admin";
 
 export type Token = {
   accessToken: string;
   refreshToken: string;
 };
 
 export type UserDTO = {
-  id: number;
+  id: string;
```

Comment:
Using the string representaiton of the MongoDB object ID. Postgres implementation will still use a number @zm2he